app/ide-desktop/lib/dashboard/src/components/AriaComponents/Text/Text.tsx

```diff
@@ -185,5 +186,5 @@ Text.Heading = React.forwardRef(function Heading(
   ref: React.Ref<HTMLHeadingElement>
```

Comment:
why as a property of the `Text` component?

Reply:
Mostly because this isn't a standalone component, but more like a preset(or shorthand for `elementType={`h${level}`} variant="h1" balance`) of the text component itself.